src/components/pages/home.jsx

```diff
@@ -1,45 +1,59 @@
 import {Link} from "react-router"
-import React from "react"
+import React, {PropTypes} from "react"
 
 import BreadCrumb from "../breadcrumb"
+import ExternalLink from "../external-link"
 
 
 const HomePage = React.createClass({
+  propTypes: {
+    countryPackageName: PropTypes.string.isRequired,
```

Comment:
Request for explanations in PRs should be a sign that some refactor / comment / documentation should be added: if it's not clear during review, it probably won't be any clearer when reading the code  :wink: